config/base/dex/numaflow-dex-server-deployment.yaml

```diff
@@ -38,9 +71,13 @@ spec:
                   name: numaflow-dex-secrets
                   key: dex-github-client-secret
       volumes:
-        - name: config
+        - name: old-config
```

Comment:
Could you add some comments on the differences between `old` and `new`? Consider using more descriptive words when naming things, words like `old` and `new` are bit vague and seem indicating we are going to deprecate the old one.

Reply:
We could change the `newConfig` name to `generatedDexConfig` since that is more accurate. `oldConfig` is not the best way to frame it either since we aren't deleting it, that is right. `connectorConfig` might be a better name.